assets/src/stories-editor/components/font-family-picker/index.js

```diff
@@ -65,14 +70,16 @@ function FontFamilyPicker( {
 				dropdownArrow={ () => '' }
 				preserveNullOptions={ true }
 				placeholder={ __( 'None', 'amp' ) }
-				showNoOptionsFound={ false }
 				displayMenu="overlay"
+				showNoOptionsFound={ false }
 				tStatusQueryTooShort={ ( minQueryLength ) =>
 					// translators: %d: the number characters required to initiate an author search.
 					sprintf( __( 'Type in %d or more characters for results', 'amp' ), minQueryLength )
 				}
 				// translators: 1: the index of thre selected result. 2: The total number of results.
```

Comment:
There's a typo in this line and I think you need to move it one line down now that you multiline'd the next attribute.

Reply:
What is the typo?

Reply:
```diff
-// translators: 1: the index of thre selected result. 2: The total number of results.
+// translators: 1: the index of the selected result. 2: The total number of results.

Reply:
Fixed https://github.com/ampproject/amp-wp/pull/3155/commits/2b91bb5ba99035055bcbb2b79dba598b46b38858